docs/detections/api/rules/signals-api-overview.asciidoc

```diff
@@ -245,3 +250,107 @@ Example response:
   "failures": []
 }
 --------------------------------------------------
+
+
+==== Apply alert tags
+
+Add and remove alerts tags on alerts.
```

Comment:
```suggestion
Add and remove tags on alerts.
```

Reply:
I actually want to keep 'alert tags' here since we use 'tags' to generally describe a set of different tags that you can add to rules and cases. I can remove the extra `alerts` at the end though.

```suggestion
Add and remove alert tags.
```

Reply:
Makes sense!